devito/propagator.py

```diff
@@ -200,7 +201,7 @@ def sympy_to_cgen(self, stencils):
                 # TODO: add support for double precision
                 self.add_local_var(name, np.float32)
                 # TODO: undo precision enforcing
-                factors.append(cgen.Assign(name, str(ccode(expr.xreplace(self._var_map))).replace("pow", "powf").replace("fabs", "fabsf")))
+                factors.append(cgen.Assign(name, str(ccode(self.time_substitutions(expr).xreplace(self._var_map))).replace("pow", "powf").replace("fabs", "fabsf")))
```

Comment:
Are you sure?

I removed it in a test on my machine and ran the tti 2D example. The generated code compiled fine so I guess there was no `s` left in the stencil. 


I think this is a left over from the initial factor implementation. A modified version of that implementation (that doesn't include this call) is already in master.